tests/test_extract.py

```diff
@@ -57,22 +57,22 @@ def django_po(self):
         """
```

Comment:
Hard-coded file names created tests that are actually independent that tested classes